game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/portsarim/grum's_gold_exchange.plugin.kts

```diff
@@ -0,0 +1,56 @@
+package gg.rsmod.plugins.content.areas.portsarim
+
+import gg.rsmod.plugins.content.mechanics.shops.CoinCurrency
+
+create_shop("Grum's Gold Exchange.", currency = CoinCurrency(), purchasePolicy = PurchasePolicy.BUY_STOCK, containsSamples = false) {
+    items[0] = ShopItem(Items.GOLD_RING, amount = 2)
```

Comment:
Default stock amount for every item in this stock should be 0 (https://discord.com/channels/1055304546521469019/1069870150322491453/threads/1103221361922560060)